src/styles/search.css

```diff
@@ -0,0 +1,26 @@
+.search-input {
+    width: 35%;
+    height: 30px;
+    border-radius: 11px;
+    font-size: 20px;
+    margin-top: 5px;
+    padding: 2px, 10px;
+    text-align: center;
+}
+
+.search-form {
+    text-align: center;
+}
+
+.search-btn {
+    height: 32px;
+    width: 80px;
+    margin-top: 5px;
```

Comment:
The padding and margin-top properties are not doing anything and can be removed. Also the text shadow property is broken, what unit are you using?